packages/components/src/tab-panel/style.scss

```diff
@@ -7,12 +7,10 @@
 	}
 }
 
-// This tab style CSS is duplicated verbatim in
-// /packages/edit-post/src/components/sidebar/settings-header/style.scss
 .components-tab-panel__tabs-item {
 	position: relative;
 	border-radius: 0;
-	height: $grid-unit-60;
+	height: $grid-unit-60 !important; // https://github.com/WordPress/gutenberg/pull/65075#discussion_r1746282734
```

Comment:
👍 I'd love it if we can clean those up at some point.

Reply:
It should be pretty simple to sweep up after the 40px becomes the default.